larray/__init__.py

```diff
@@ -6,5 +6,6 @@
 from larray.example import *
 from larray.extra import *
 from larray.viewer import *
+import larray.random
```

Comment:
because we have to draw the line somewhere. We cannot indefinitely add stuff to the larray namespace. Currently, there are only a few functions, but this is bound to grow. In LIAM2, I added all random functions to directly in the top-level namespace but I regretted that decision when I added support for most the random functions from numpy.